model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/Vcf.java

```diff
@@ -0,0 +1,45 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class Vcf
```

Comment:
Same suggestion as the previous classes, make the class members `private`, and gather them on the top.